### PR TITLE
pin pillow

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: sunpy_version_pins.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pillow!=7.1.0
```
Comment (Cadair) [inline sunpy_version_pins.txt:1]:
```suggestion
pillow!=7.1.0
pytest<5.4
```

Comment (nabobalis) [reply sunpy_version_pins.txt:1]: Are we broken with pytest?

Comment (Cadair) [reply sunpy_version_pins.txt:1]: doctestplus is, it causes a warning.

Comment (nabobalis) [reply sunpy_version_pins.txt:1]: A warning or an error?

Comment (Cadair) [reply sunpy_version_pins.txt:1]: warning.

Comment (nabobalis) [reply sunpy_version_pins.txt:1]: Is it worth pinning for a warning?

**Pull request merged**